doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -997,9 +997,17 @@ Generation of induction principles with ``Scheme``
   :n:`Minimality for @reference Sort @sort_family`
     Defines a non-dependent elimination principle more natural for inductively defined relations.
 
+  :n:`Boolean Equality for @reference`
+     Tries to generate a Boolean equality.
+     If :token:`reference` involves independent constants or other
+     inductive types, it is recommended to define their equality
+     first.
+
```

Comment:
Much of this repeats the description for Equality below, let's integrate it below.